games-puzzle/dynamate/dynamate-1.0.0.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A game about merging bombs of different colors"
+DESCRIPTION="Dynamate is a game where you try to remove all colored dots \
+from the screen. There are specific rules as to how you can manipulate \
+each dot, and what each different colored dot does.
+
+Dynamate is a single-player game with many different levels."
+HOMEPAGE="https://github.com/HaikuArchives/Dynamate"
+COPYRIGHT="2014 Adrien Destugues
+1995 Jonas Norberg, Signe Norberg and Bjorn Kalzen"
+LICENSE="MIT"
+SOURCE_URI="git+https://github.com/HaikuArchives/Dynamate.git#e870f3c3026739831b749318d58826b0e4d874d7"
+REVISION="1"
+
+ARCHITECTURES="x86 ?x86_gcc2 ?x86_64"
+
+PROVIDES="
+	dynamate = $portVersion
+	app:dynamate = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Dynamate
+
+	cp -r DynaMate/* $appsDir/Dynamate/
```

Comment:
mkdir -p $appsDir
cp -r DynaMate $appsDir

This should also work and is shorter